pom.xml

```diff
@@ -3053,6 +3053,17 @@
         <version>${jackson1.version}</version>
       </dependency>
       <!-- jackson 2 (used by addon) -->
+      <dependency>
+        <groupId>org.codehaus.woodstox</groupId>
+        <artifactId>stax2-api</artifactId>
+        <version>4.2</version>
+        <exclusions>
+          <exclusion>
+            <artifactId>stax-api</artifactId>
+            <groupId>javax.xml.stream</groupId>
+          </exclusion>
+        </exclusions>
```

Comment:
I don't think that this exclusion is needed, `stax2-api` doesn't depend on `stax-api`

Reply:
Removed also an exclusion of stax in woodstox core I was adding but not needed